myia/frontends/pytorch.py

```diff
@@ -117,17 +150,19 @@ def _to_abstract(self, v: torch.nn.Module, **kwargs):
         '__call__': getattr(type(v), 'forward'),
         '__sub__': mod_sub,
     }
-    attrs = {}
-    for var_k, var_v in vars(v).items():
-        if var_k not in blacklist:
-
-            # TODO: Remove "(isinstance(v, torch.nn.Sequential) and"
-            #       once Alias PR ready
-            # TODO: Remove rest of if statement once Dict supports empty Dict
-            if var_k not in ('_parameters', '_modules') or \
-                    (isinstance(v, torch.nn.Sequential) and
-                     var_v != OrderedDict()):
-                attrs[var_k] = self(var_v, **kwargs)
+    fields = {}
+    for var_k in dir(v):
+        if (var_k not in blacklist) or (var_k in ('_parameters', '_modules')):
```

Comment:
It would be better to simply remove `_parameters` and `_modules` from the blacklist when it is created.

Reply:
This portion of if statement will be removed when alias for classes/module is supported in next PR. It's easier for me remember to remove it when it looks awkward in if statement (like it currently does).

Reply:
Okay, add a comment to explain this.